aws-rds-globalcluster/src/main/java/software/amazon/rds/globalcluster/Translator.java

```diff
@@ -18,22 +18,13 @@
 public class Translator {
 
   static CreateGlobalClusterRequest createGlobalClusterRequest(final ResourceModel model) {
-    return CreateGlobalClusterRequest.builder()
-            .engine(model.getEngine())
-            .engineVersion(model.getEngineVersion())
-            .storageEncrypted(model.getStorageEncrypted())
-            .deletionProtection(model.getDeletionProtection())
-            .globalClusterIdentifier(model.getGlobalClusterIdentifier())
-            .sourceDBClusterIdentifier(model.getSourceDBClusterIdentifier())
-            .storageEncrypted(model.getStorageEncrypted())
-            .build();
+    return createGlobalClusterRequest(model, null);
```

Comment:
why?

Reply:
the code does absolutely the same as the other method.

Reply:
do we need this code though?

Reply:
its' used in CreateHandler.